integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -143,6 +143,14 @@ msgstr "PPT-Dokument"
 msgid "PPTX document"
 msgstr "PPTX-Dokument"
 
+#: cms/constants/calendar.py
+msgid "Event not from an external calendar"
+msgstr "Veranstaltungen ohne externen Kalender"
+
+#: cms/constants/calendar.py
+msgid "Event from an external calendar"
+msgstr "Veranstaltungen aus externen Kalender"
+
```

Comment:
```suggestion
#: cms/constants/calendar.py
msgid "Event does not come from an external calendar"
msgstr "Veranstaltung kommt nicht aus einem externen Kalender"

#: cms/constants/calendar.py
msgid "Event comes from an external calendar"
msgstr "Veranstaltungen kommt aus einem externen Kalender"

```

(Nit / does not have to be exactly this)

"Veranstaltungen ohne externen Kalender" to me sounds too much like "Verantsaltungen, bei denen der externe Kalender fehlt"